### DISCUSSION TITLE
Why not request for merging with official Signal repos

### DISCUSSION THREAD

**Discussion by tintad** [signalstickers/signalstickers#912] [Q&A]
Hello everyone

I'm just wondering why we don't request for an official merge with Signal sticker store so that it's easier for new users to find stickers

**Comment by romainricard** [comment]
I'm not sure about "signal sticker store", what do you mean? Afaik there's no official sticker store

**Reply by tintad** [reply]
I mean the one that you can download directly from the app. Usually every messaging app has a place for users to download stickers directly. If we could do the same it would be very easy for new users to find and downloads and designers to share their work also
